README.rst

```diff
@@ -0,0 +1,164 @@
+PennyLane-Qulacs Plugin
+######################
+
+.. image:: https://github.com/XanaduAI/pennylane-cirq/blob/master/doc/_static/puzzle_qulacs.png?raw=true
+    :align: center
+    :width: 200px
+    :target: javascript:void(0);
+
+|
+
+.. header-start-inclusion-marker-do-not-remove
+
+The PennyLane-Qulacs plugin integrates the Qulacs quantum computing framework with PennyLane's
+quantum machine learning capabilities.
+
+`PennyLane <https://pennylane.readthedocs.io>`__ is a cross-platform Python library for quantum machine
+learning, automatic differentiation, and optimization of hybrid quantum-classical computations.
+
+`Qulacs <https://github.com/qulacs>`__ is a software library for quantum computing, written in C++ and with GPU support.
+
+.. header-end-inclusion-marker-do-not-remove
+
+The plugin documentation can be found here: `<https://pennylane-qulacs.readthedocs.io/en/latest/>`__.
+
+Features
+========
+
+* Provides access to Qulacs' simulator backend via the ``qulacs.simulator`` device
+
+* Support for all PennyLane core functionality
+
+.. benchmarks-start-inclusion-marker-do-not-remove
+
+Benchmarks
+==========
+
```

Comment:
The benchmarks are from the original authors, probably outdated. I still find them quite interesting. Can we specify the number of workers in the device?

Reply:
I think they're nice to add as well. Maybe we could just update them later at some point. :slightly_smiling_face: What do you mean by workers?

Reply:
From the plot it seems qulacs can be run with a custom number of "nodes" or "cores" on a CPU...that would be a crucial feature! :)

Reply:
Ah, from a quick glance it seems like Qulacs uses OpenMP for parallelisation, meaning that it might be possible to switch between the number of threads by setting the environment variable `OMP_NUM_THREADS` to a value corresponding to the number of threads that should be used.

Not sure if there's another way to do this though.